microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -57,6 +57,8 @@ def ltp_lite(
         # parse variables
         tests = variables.get("ltp_test", "")
         skip_tests = variables.get("ltp_skip_test", "")
+        use_block_drive = variables.get("use_block_drive", True)
+        use_temp_dir = variables.get("use_temp_dir", False)
```

Comment:
What's the reason need this? It's confusing, people may think it applies to all test cases.

Reply:
This is not related to the use of os disk. It's a parameter in the LTP test. We don't need to enable it in our tests but the WSL testing is being done with this parameter

Reply:
Do you know why WSL need this parameter? Can we always put the results in tmp or home folder?

Reply:
The parameters will make test cases complex to configure and run. Even we provide the default values, the unneeded parameters will be confusing.

Reply:
Fixed! Remove the temporary directory variable